dhcpv6/option_fqdn.go

```diff
@@ -0,0 +1,40 @@
+package dhcpv6
+
+import (
+        "fmt"
```

Comment:
Indentation looks wrong here, "fmt" and "github.com/u-root/u-root/pkg/uio" do not look aligned.